### DISCUSSION TITLE
Coastline Vector Questions

### DISCUSSION THREAD

**Discussion by 2320sharon** [SatelliteShorelines/CoastSeg#21] [Q&A]
# Coastline Vector Accuracy Question
The coastline vector from https://geodata.lib.berkeley.edu/catalog/stanford-xv279yj9196 is not super accurate to the actual coastline when at a zoomed-in level as shown below. Is this going to be okay if we are using it to measure shoreline distances?

![coastline vector](https://user-images.githubusercontent.com/61564689/148292653-9c7559e9-ceb1-4bdf-acdf-92dd384fe3bd.png)

**Comment by kvos** [comment]
@2320sharon there is also the 10m resolution Natural Earth coastline which is great https://www.naturalearthdata.com/downloads/10m-physical-vectors/

**Comment by ddbuscombe** [comment]
Yes, we recently used these shoreline data for a different project. See https://github.com/C-GRASP/AnalysisNotebooks/blob/main/sample_compute_distance_to_shore/sample_compute_distance_to_shore.ipynb

@2320sharon perhaps you could pull in both shoreline vectors and compare them in troublesome locations?

The advantage to this method (if the Natural Earth data are superior to the stanford data) is that the data are built into cartopy `coast = cartopy.io.shapereader.natural_earth(resolution='10m',category='physical', name='coastline')`
